### DISCUSSION TITLE
如何开发插件并在本地调试？

### DISCUSSION THREAD

**Discussion by axiangcoding** [koishijs/koishi#1276] [Q&A] [answered]
根据文档 https://koishi.chat/zh-CN/guide/develop/workspace.html#%E5%88%9B%E5%BB%BA%E6%96%B0%E6%8F%92%E4%BB%B6 的描述
> 以 [开发模式](https://koishi.chat/zh-CN/guide/develop/script.html#%E5%BC%80%E5%8F%91%E6%A8%A1%E5%BC%8F) 重新运行你的项目，点击右上角的「添加插件」按钮，选择你刚才创建的插件名称，你会立即在网页控制台的配置界面中看到 example 插件。只需点击启用，你就可以实现与机器人的对话了：

那么这个“右上角的「添加插件」按钮”在哪里...

**Comment by ilharp** [comment, marked as answer]
![image](https://github.com/koishijs/koishi/assets/20179549/37061f97-d5ac-49b9-b1da-ef08ee559f5d)

入门 Koishi 请先阅读入门文档。

[添加更多插件 | Koishi](https://koishi.chat/zh-CN/manual/usage/market.html#%E6%B7%BB%E5%8A%A0%E6%9B%B4%E5%A4%9A%E6%8F%92%E4%BB%B6)

**Reply by axiangcoding** [reply]
谢谢回复，发现是插件名称冲突导致了自定义插件没搜到，改名后就正常了
